flows/actions/reply.go

```diff
@@ -51,6 +54,15 @@ func (a *ReplyAction) Execute(run flows.FlowRun, step flows.Step) error {
 		return nil
 	}
 
-	run.AddEvent(step, events.NewSendMsgToContact(run.Contact().UUID(), text, a.Attachments))
+	urns := run.Contact().URNs()
+	if a.AllURNs && len(urns) > 0 {
+		for i := range urns {
```

Comment:
This may be a question of how we expect this to work for contacts that have no URNs but I think the more go-ing way of doing this would probably be:

```go
if a.AllURNS {
  for _, urn := range run.Contact().URNS() {
     run.AddEvent(step, events.NewSendMsgToURN(urn, text, a.Attachments))
  }
} else {
...
}
```